src/coreclr/scripts/superpmi_asmdiffs_checked_release_setup.py

```diff
@@ -0,0 +1,247 @@
+
+    coreclr_args.verify(args,
+                        "arch",
+                        lambda unused: True,
+                        "Unable to set arch")
+
+    coreclr_args.verify(args,
+                        "source_directory",
+                        lambda source_directory: os.path.isdir(source_directory),
+                        "source_directory doesn't exist")
+
+    coreclr_args.verify(args,
+                        "product_directory",
+                        lambda product_directory: os.path.isdir(product_directory),
+                        "product_directory doesn't exist")
+
+    coreclr_args.verify(args,
+                        "release_directory",
+                        lambda release_directory: os.path.isdir(release_directory),
+                        "release_directory doesn't exist")
+                        
+    return coreclr_args
+
+
+def match_jit_files(full_path):
+    """ Match all the JIT files that we want to copy and use.
+        Note that we currently only match Windows files, and not osx cross-compile files.
+        We also don't copy the "default" clrjit.dll, since we always use the fully specified
+        JITs, e.g., clrjit_win_x64_x64.dll.
+    """
+    file_name = os.path.basename(full_path)
+
+    if file_name.startswith("clrjit_") and file_name.endswith(".dll") and file_name.find("osx") == -1:
+        return True
+
+    return False
+
+
+def match_superpmi_tool_files(full_path):
+    """ Match all the SuperPMI tool files that we want to copy and use.
+        Note that we currently only match Windows files.
+    """
+    file_name = os.path.basename(full_path)
+
+    if file_name == "superpmi.exe" or file_name == "mcs.exe":
+        return True
+
+    return False
+
+
+def main(main_args):
+    """Main entrypoint: Prepare the Helix data for SuperPMI asmdiffs checked release Azure DevOps pipeline.
+
+    The Helix correlation payload directory is created and populated as follows:
+
+    <source_directory>\payload -- the correlation payload directory
+        -- contains the *.py scripts from <source_directory>\src\coreclr\scripts
+        -- contains superpmi.exe, mcs.exe from the target-specific build
+    <source_directory>\payload\base
+        -- contains the Checked JITs
+    <source_directory>\payload\diff
+        -- contains the Release JITs
+    <source_directory>\payload\jit-analyze
+        -- contains the self-contained jit-analyze build (from dotnet/jitutils)
+    <source_directory>\payload\git
+        -- contains a Portable ("xcopy installable") `git` tool, downloaded from:
+        https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/git/Git-2.32.0-64-bit.zip
+        This is needed by jit-analyze to do `git diff` on the generated asm. The `<source_directory>\payload\git\cmd`
+        directory is added to the PATH.
+        NOTE: this only runs on Windows.
```

Comment:
Because we don't need jit-analyze, we don't need this, either.

Reply:
Removed.